scripts/stage-release.sh

```diff
@@ -172,6 +172,13 @@ updateVersions "${RELEASE_VERSION}" "${DEVELOPMENT_VERSION}"
 ${SED} -i -e "s_##\s${RELEASE_VERSION//./\\.}_## SNAPSHOT\n## ${RELEASE_VERSION//./\\.}_g" CHANGELOG.md
 
 git add 'CHANGELOG.md'
+
+# bump the reference version in kroxylicious-api
+mvn -q -B -f kroxylicious-api/pom.xml versions:set-property -Dproperty="ApiCompatability.ReferenceVersion" -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false
```

Comment:
Yes of course this is simpler than I had been thinking when this script runs we don't care about the previous release we are setting a new baseline. 

That will hold true even when we go to 1.0.0, right? As we will still maintain the same binary compat rules with every minor/patch bump

Reply:
Yes I think it will hold true for 1.0.0+, though maybe we will revisit versioning and the deprecation policy at that point (we could decide to use semver more fully)